library/network_connections.py

```diff
@@ -386,6 +386,27 @@ def ifcfg_create(
                 " ".join(configured_features),
             )
 
+        ethtool_coalesce = connection["ethtool"]["coalesce"]
+        configured_coalesce = []
+        for coalesce, setting in ethtool_coalesce.items():
+            if setting is not None:
+                if isinstance(setting, bool):
+                    configured_coalesce.append(
+                        "%s %u" % (coalesce.replace("_", "-"), setting)
+                    )
+                else:
+                    configured_coalesce.append(
+                        "%s %s" % (coalesce.replace("_", "-"), setting)
+                    )
```

Comment:
Currently, both branches for the condition differ in only one character. It seems that it could be simplified as

```suggestion
                if isinstance(setting, bool):
                    setting = int(setting)
                configured_coalesce.append(
                        "%s %s" % (coalesce.replace("_", "-"), setting)
                    )
```

to explicitly show the difference for bool. Whatdo you think?

Reply:
Changed, thanks!